### ISSUE TITLE
Manifest file reference in the Sway Book has a wrong key listed: `implicit-std` is spelled as `implicit_std`

### ISSUE THREAD

**Issue by kayagokalp** [FuelLabs/sway#1779]
`implicit_std` should be changed to `implicit-std` in the manifest reference section of the Sway Book.

**Comment by adlerjohn** [comment]
Cargo uses dashes instead of underscores indeed: https://doc.rust-lang.org/cargo/reference/manifest.html

**Comment by mohammadfawaz** [comment]
Yeah this is a documentation bug only. `forc` expects a dash as well.
